source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -562,6 +562,23 @@ The :guilabel:`Security` section displays TLS certificate settings for the MinIO
 
        MinIO also supports uploading Certificate Authority certificates for validating client certificates minted by that CA.
 
+.. admonition:: Supported Secret Types
+   :class: note
+    
+   MinIO supports three types of :kube-docs:`secrets in Kubernetes <concepts/configuration/secret/#secret-types>`.
+       
+   #. opaque
```

Comment:
Are _opaque_ and _tls_ here keywords, of a sort that might get `inline code` ?

Reply:
They are specific types in the linked K8s docs list.